Explorer/Assets/DCL/UserInAppInitializationFlow/StartupOperations/SetOverrideStartingParcelOperation.cs

```diff
@@ -0,0 +1,47 @@
+using Cysharp.Threading.Tasks;
+using DCL.AsyncLoadReporting;
+using DCL.FeatureFlags;
+using ECS.SceneLifeCycle.Realm;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+using Utility.Types;
+
+namespace DCL.UserInAppInitializationFlow.StartupOperations
+{
+    public class SetOverrideStartingParcelOperation : IStartupOperation
+    {
+        private readonly IRealmNavigator realmNavigator;
+        private readonly Vector2Int startParcel;
+        private readonly FeatureFlagsCache featureFlagsCache;
+        private readonly bool isLocalDevelopment;
+
+        public SetOverrideStartingParcelOperation(
+            IRealmNavigator realmNavigator,
+            Vector2Int startParcel,
+            FeatureFlagsCache featureFlagsCache,
+            bool isLocalDevelopment)
+        {
+            this.realmNavigator = realmNavigator;
+            this.startParcel = startParcel;
+            this.featureFlagsCache = featureFlagsCache;
+            this.isLocalDevelopment = isLocalDevelopment;
+        }
+
+        public async UniTask<Result> ExecuteAsync(AsyncLoadProcessReport report, CancellationToken ct)
+        {
+            string? parcelToTeleportOverride = null;
+
+            bool overrideStartingParcel = !isLocalDevelopment &&
+                                          startParcel == Vector2Int.zero &&
```

Comment:
@fcolarich will this work if I want to go to 0,0; but the start parcel is elsewhere? 

Wouldnt it just take me to the elsewhere place?